src/DropdownMenu.js

```diff
@@ -2,22 +2,32 @@ import classNames from 'classnames';
 import keycode from 'keycode';
 import React from 'react';
 import ReactDOM from 'react-dom';
+import RootCloseWrapper from 'react-overlays/lib/RootCloseWrapper';
 
-import { prefix } from './utils/bootstrapUtils';
+import { bsClass, getClassSet, omitBsProps, prefix }
+  from './utils/bootstrapUtils';
 import createChainedFunction from './utils/createChainedFunction';
 import ValidComponentChildren from './utils/ValidComponentChildren';
 
-import RootCloseWrapper from 'react-overlays/lib/RootCloseWrapper';
+const propTypes = {
+  open: React.PropTypes.bool,
+  pullRight: React.PropTypes.bool,
+  onClose: React.PropTypes.func,
+  labelledBy: React.PropTypes.oneOfType([
+    React.PropTypes.string, React.PropTypes.number,
+  ]),
+  onSelect: React.PropTypes.func,
+};
+
+const defaultProps = {
+  bsRole: 'menu',
+  pullRight: false,
+};
 
 class DropdownMenu extends React.Component {
   constructor(props) {
     super(props);
 
-    this.focusNext = this.focusNext.bind(this);
```

Comment:
DropdownMenu calls it as `this.menu.focusNext()` though, which doesn't require binding. It's a pretty standard caveat with class-like objects in JS, so I figure why make it any different here?